coaster-nn/src/frameworks/native/mod.rs

```diff
@@ -886,6 +888,7 @@ impl<T> Rnn<T> for Backend<Native>
     }
 }
 
+#[cfg(feature = "native")]
```

Comment:
iirc, I tried this too, but it is pretty hard to envision a use case without having a native tensor first.